src/components/dialog/dialog/dialog.stories.ts

```diff
@@ -0,0 +1,483 @@
+import { withActions } from '@storybook/addon-actions/decorator';
+import { userEvent, within } from '@storybook/testing-library';
+import type { InputType } from '@storybook/types';
+import type {
+  Meta,
+  StoryObj,
+  ArgTypes,
+  Args,
+  Decorator,
+  StoryContext,
+} from '@storybook/web-components';
+import isChromatic from 'chromatic';
+import type { TemplateResult } from 'lit';
+import { html } from 'lit';
+import { styleMap } from 'lit/directives/style-map.js';
+
+import { waitForComponentsReady } from '../../../storybook/testing/wait-for-components-ready';
+import { waitForStablePosition } from '../../../storybook/testing/wait-for-stable-position';
+import { sbbSpread, type Breakpoint } from '../../core/dom';
+import sampleImages from '../../core/images';
+import type { TitleLevel } from '../../title';
+import { SbbDialogTitleElement } from '../dialog-title';
+
+import { SbbDialogElement } from './dialog';
+import readme from './readme.md?raw';
+
+import '../../button';
+import '../../link';
+import '../../title';
+import '../../form-field';
+import '../../image';
+import '../../action-group';
+import '../dialog-content';
+import '../dialog-actions';
+
+// Story interaction executed after the story renders
+const playStory = async ({ canvasElement }: StoryContext): Promise<void> => {
+  const canvas = within(canvasElement);
+
+  await waitForComponentsReady(() =>
+    canvas.getByTestId('dialog').shadowRoot!.querySelector('.sbb-dialog'),
+  );
+
+  await waitForStablePosition(() => canvas.getByTestId('dialog-trigger'));
+
+  const button = canvas.getByTestId('dialog-trigger');
+  await userEvent.click(button);
+};
+
+const level: InputType = {
+  control: {
+    type: 'inline-radio',
+  },
+  options: [1, 2, 3, 4, 5, 6],
+  table: {
+    category: 'Title',
+  },
+};
+
+const backButton: InputType = {
+  control: {
+    type: 'boolean',
+  },
+  table: {
+    category: 'Title',
+  },
+};
+
+const hideOnScroll: InputType = {
+  control: {
+    type: 'select',
+  },
+  options: ['zero', 'micro', 'small', 'medium', 'large', 'wide', 'ultra'],
```

Comment:
```suggestion
  options: ['off', 'zero', 'micro', 'small', 'medium', 'large', 'wide', 'ultra'],
```
Should we add an 'off' value?
And in the story title template something like `hide-on-scroll=${hideOnScroll !== 'off' ? hideOnScroll : nothing}`

Reply:
I see your point but I'm afraid it'd be confusing for the consumer who might think that `off` would be a possible value for the property.

Reply:
true... can't see a good solution that makes everyone happy :/